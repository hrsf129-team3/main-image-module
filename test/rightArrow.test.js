import React from 'react';
import RightArrow from '../client/src/components/rightArrow.jsx';

describe('Right Arrow', () => {
  it('should render right arrow component', () => {
    const component = shallow(<RightArrow />);
    expect(component).toMatchSnapshot();
  });
})


