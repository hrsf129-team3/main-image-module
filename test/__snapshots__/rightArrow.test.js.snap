// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Right Arrow should render right arrow component 1`] = `ShallowWrapper {}`;
